packages/ember-metal/lib/is_proxy.js

```diff
@@ -1,7 +1,10 @@
-import { symbol } from 'ember-utils';
-
-export const IS_PROXY = symbol('IS_PROXY');
+import { peekMeta } from './meta';
 
 export function isProxy(value) {
-  return typeof value === 'object' && value && value[IS_PROXY];
+  if (typeof value === 'object' && value) {
+    let meta = peekMeta(value);
+    return typeof meta === 'object' && meta !== null && meta.isProxy();
```

Comment:
Correct, we could simplify to just be `return meta !== undefined && meta.isProxy()`  or something like that.  

@chadhietala - Sorry for the wild-goose chase (I realize this snippet came from my suggestion in a prior comment).